uwblueprint/supportive-housing
# Overflow gets cancelled on login | BUG

## Implementation description
* I've had this weird bug when I logout and log back in that basically freezes the screen and blocks scrolling on the actual page and only allows it on the table. I've been able to get past this with refreshing the page but that is not very intuitive for the user.



## Steps to test
1. See if you can reproduce the bug on your end, press log out and then sign back in. Does the scroll lock?
2. If so, does this fix it for you?



## What should reviewers focus on?
* I think this fixes it on my side not sure if it's a universal fix tho
* Also are there any risks to setting this here? I'm not too sure how it affects the rest of the app.


## Checklist
